01-node-tutorial/answers/practice1.js

```diff
@@ -0,0 +1,24 @@
+const {writeFile} = require('fs').promises
```

Comment:
Did you add these on your own initiative? Awesome! I love it when students do some extra tinkering; I strongly encourage you to continue!